lib/restforce/db/cleaner.rb

```diff
@@ -40,15 +40,11 @@ def invalid_salesforce_ids
       #
       # Returns an Array of IDs.
       def all_salesforce_ids
-        all_ids = []
-
         @mapping.unscoped do |map|
           @runner.run(map) do |run|
-            run.salesforce_instances { |instance| all_ids << instance.id }
+            run.salesforce_instances.map(&:id)
```

Comment:
It occurs to me that `all_salesforce_ids` is just calling `valid_salesforce_ids` inside an `unscoped` block, and could be represented as such.